src/VisualStudio/Core/Def/Implementation/AnalyzerDependency/AnalyzerDependencyCheckingService.cs

```diff
@@ -80,48 +80,57 @@ internal sealed class AnalyzerDependencyCheckingService
 
             var conflicts = results.Conflicts;
             var missingDependencies = results.MissingDependencies;
+            var projects = _workspace.DeferredState.ProjectTracker.ImmutableProjects;
 
-            foreach (var project in _workspace.DeferredState.ProjectTracker.ImmutableProjects)
+            // Report conflict resolution diagnostics on a background thread.
+            // This is done to avoid blocking the UI thread for attribute binding to determine diagnostic suppression.
+            // See https://github.com/dotnet/roslyn/issues/20476 for further details.
+#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
```

Comment:
❕ There is no need to disable this warning. Since the enclosing method is `async`, you can `await` this without a problem.